localstack/services/apigateway/integration.py

```diff
@@ -119,6 +119,23 @@ def apply_response_parameters(
                 response.headers[header_name] = value.strip("'")
         return response
 
+    @classmethod
+    def render_template_selection_expression(cls, invocation_context: ApiInvocationContext):
```

Comment:
nit: I'm not too fond of mixing websockets logic in community as we've talked offline, but I believe for now, it's the quickest way, as it's already done for kinesis anyway. We will refactor this at some point!